packages/opentelemetry-exporter-prometheus/src/prometheus.ts

```diff
@@ -69,10 +69,13 @@ export class PrometheusExporter implements MetricExporter {
       config.endpoint || PrometheusExporter.DEFAULT_OPTIONS.endpoint
     ).replace(/^([^/])/, '/$1');
 
-    if (config.startServer || PrometheusExporter.DEFAULT_OPTIONS.startServer) {
+    if (config.startServer) {
       this.startServer(callback);
-    } else if (callback) {
-      callback();
+    } else {
+      this._logger.debug(
```

Comment:
If the user specifically puts in a `false` config, we don't need to log. In this case it is not an unexpected or error condition.

Reply:
Reasonable. I'll remove this log.